docs/build/html/_sources/contribute.rst.txt

```diff
@@ -35,7 +35,7 @@ forward on all code in PyU4V.
 
 2. Use ``.format()`` for string concatenation.
 
-.. code-block::
+.. code-block:: text
```

Comment:
Code type should be python here

Reply:
It complained about it when I put in 'Python'.  Let me try it again